.gitpod.yml

```diff
@@ -0,0 +1,15 @@
+image:
+  file: .gitpod.Dockerfile
+
+ports:
+  - port: 8000
+    onOpen: open-preview
+
+tasks:
+  - init: |
+      ./tools/update.sh
```

Comment:
It wold be better to make this be:
```
- init: |
    yarn update
  command: yarn dev-headless
```

Should `yarn dev-webui` also be run to allow for editing of the webui code? If so the open port above should be changed to `3000`

Reply:
Currently it doesn't work, since every port on Gitpod is exposed on a different host (example: port 8000 `https://3000-matteogheza-companion-[...].gitpod.io/` and port 3000 `https://3000-matteogheza-companion-[...].gitpod.io/`).
I tried to run this (after exporting the `REACT_APP_SERVER_URL` env), but it doesn't work for CORS reasons.

![chrome_oFbWTpUVpn](https://user-images.githubusercontent.com/27729549/163201570-dd2d59ce-3a72-4564-a75c-12fc2cd6f51d.png)